Source/WebCore/dom/Range.cpp

```diff
@@ -311,6 +311,14 @@ ExceptionOr<RefPtr<DocumentFragment>> Range::processContents(ActionType action)
 
     RefPtr<Node> commonRoot = commonAncestorContainer();
     ASSERT(commonRoot);
+    
+    // For extractContents(), abort early if there's a doctype in range.
```

Comment:
This comment just repeats what the code says. Please remove.

Reply:
Done, thanks!